js/src/views/edit-attribute-field.js

```diff
@@ -71,6 +72,10 @@ var editAttributeField = Backbone.View.extend( {
 			this.setValue( $el.filter(':checked').first().val() );
 		} else if ( 'checkbox' === this.model.attributes.type ) {
 			this.setValue( $el.is( ':checked' ) );
+		}  else if ( 'range' === this.model.attributes.type ) {
+			var rangeId =  '#' + e.target.id + '_indicator';
+			var rangeValue = e.target.value;
+			document.querySelector( rangeId ).value = rangeValue;
```

Comment:
You still need to call `this.setValue()`, otherwise the underlying model won't be updated and won't save properly.